mods/flowers/init.lua

```diff
@@ -46,6 +51,7 @@ local function add_simple_flower(name, desc, box, f_groups)
 		paramtype = "light",
 		walkable = false,
 		buildable_to = true,
+		stack_max = 99,
```

Comment:
Unrelated changes. Did you copy & paste the files from an old MTG version? Please use rebase if you run into conflicts.

Reply:
should have been the stable-5 branch but I may be wrong

Reply:
Yes, the `master` branch would be up-to-date. There were already a few little changes since stable-5.

Reply:
I made fresh start with a pull from master but copied changed files from my earlier stable-5 work into that. Obviously, my scan and check to repair the deviations was a partial failure.

I will learn from this and seek to improve. Thanks again.